packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -519,16 +542,16 @@ class Treemap extends StatelessWidget {
   /// Selecting this widget will trigger a re-root of the tree
   /// to the associated [TreemapNode].
   ///
-  /// The default value for newRoot is [rootNode].
+  /// The default value for newRoot is [widget.rootNode].
   Widget buildSelectable({@required Widget child, TreemapNode newRoot}) {
-    newRoot ??= rootNode;
+    newRoot ??= widget.rootNode;
     return Tooltip(
```

Comment:
if we have a tooltip above, can we remove this tooltip?

Reply:
buildSelectable is used for title bars and cells on the first level without any children. In both cases, there is no Tooltip above the widgets since we're not painting them.